ci_server/server.py

```diff
@@ -6,20 +6,22 @@ class CIServer(BaseHTTPRequestHandler):
     TODO: implement the CI server, this is just a placeholder.
     """
 
-    def do_GET(self):
+    def do_get(self):
         if self.path == "/":
             self.path = "../static/ci_server/index.html"
         try:
             file = open(self.path[1:]).read()
             self.send_response(200)
-        except:
+        except FileNotFoundError:
             file = "File not found"
             self.send_response(404)
+
         self.end_headers()
         self.wfile.write(bytes(file, "utf8"))
+        file.close() # Remember to close files.
```

Comment:
This should result in an error, as `file` is actually of type string or bytes in this code example, not a file pointer.  Closing the file is not necessary (although good practice) as pythons garbage collector will do it for us. But as it looks like a bug I think it is best removed

Reply:
Thank you for pointing that out. I will correct it.

Reply:
This has now been corrected.